eng/pipelines/templates/steps/select-feed.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  # Publish to https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-net
+  DevOpsFeedId: '29ec6040-b234-4e31-b139-33dc4287b756/fa8c16a3-dbe0-4de2-a297-03065ec1ba3f'
```

Comment:
Should we have this in some global variable file that we can override for the private repo?

Reply:
I was considering it, but how it gets used here in a param is too soon to rely on global vars. Basically, this is the default. The only other place besides this file is where it's used in a pipeline that now includes this template, and I would've removed it except it's used unconditionally in one other place in that file.

That said, it's by far the only duplication we have. Unfortunate, but not uncommon.